### PR TITLE
fw 103 and 102 homework

### PR DESCRIPTION
**Pull request by choiii-darren**
firmware 102 and 103 onboarding homework for Darren Choi

### CODE REVIEW

File: gdz [hunk 1]
```diff
@@ -0,0 +1,105 @@
+  help[m
```
Comment (AveryChiu64) [inline gdz:1]: huh what is this file?

File: projects/hello_world/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!--
```
Comment (AveryChiu64) [inline projects/hello_world/README.md:1]: The 'Hello World' project should not be included with the PR

File: projects/hello_world/src/main.c [hunk 3]
```diff
@@ -0,0 +1,97 @@
+#include "../../../libraries/ms-common/inc/adc.h"
```
Comment (AveryChiu64) [inline projects/hello_world/src/main.c:1]: Hmm this is strange, you should be able to just do `#include "adc.h"`

File: projects/tutorial_board_adc/src/main.c [hunk 4]
```diff
@@ -1,4 +1,4 @@
-#include "adc.h"
+#include "../../../libraries/ms-common/inc/adc.h"
```
Comment (AveryChiu64) [inline projects/tutorial_board_adc/src/main.c:1]: This change should not be included as part of the PR. 
When we make PR's we are saying "I would like these changes to be added to the codebase and for everyone else to see it once it has been merged"

Though this is just the homework, its still best to keep PR's clean

File: projects/hello_world/src/main.c [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+typedef struct Counters {
+  uint8_t counter_a;
+  uint8_t counter_b;
+} Counters;
+
+static void soft_counter_callback(const SoftTimerId timer_id, void *context) {
+  Counters *counter1 = (Counters *)context;
+  // LOG_DEBUG("counter1 address: %p \n", &counter1);
```
Comment (AveryChiu64) [inline projects/hello_world/src/main.c:62]: Remove code used for testing when submitting a PR

File: projects/hello_world/src/main.c [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+  prv_init_gpio_pins();
+
+  adc_init(ADC_MODE_SINGLE);
+
+  AdcChannel sensor_channel = NUM_ADC_CHANNELS;
+  adc_get_channel(sensor_addresses[A6_SENSOR], &sensor_channel);
+  adc_set_channel(sensor_channel, true);
+
+  LOG_DEBUG("Hello world!\n");
```
Comment (AveryChiu64) [inline projects/hello_world/src/main.c:86]: Please remove this

**Review by AveryChiu64 — COMMENTED** [review]
Please separate the firmware 102 and 103 homework into separate projects. They should each be under their own folder, not hello_world

File: projects/hello_world/src/main.c (hunk 3 above)
Comment (choiii-darren) [reply projects/hello_world/src/main.c:1]: my vscode had an interesting file also called adc.h so i had to include the direct path

**Pull request closed without merging**